tests/plugins/test_push_floating_tags.py

```diff
@@ -0,0 +1,456 @@
+             OTHER_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+         },
+         'x86_64': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+         }
+     },
+     None),
+    ("simple_oci",
+     [REGISTRY_V2, OTHER_V2], OCI_RESULTS, 'oci',
+     ['namespace/httpd:2.4-1'],
+     {
+         'ppc64le': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+         },
+         'x86_64': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+         }
+     },
+     None),
+    ("multi_v2",
+     [REGISTRY_V2, OTHER_V2], V2_RESULTS, 'v2',
+     ['namespace/httpd:2.4-1', 'namespace/httpd:latest'],
+     {
+         'ppc64le': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+         },
+         'x86_64': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+         }
+     },
+     None),
+    ("No tags",
+     [REGISTRY_V2, OTHER_V2], V2_RESULTS, 'v2',
+     None,
+     {
+         'ppc64le': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+         },
+         'x86_64': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+         }
+     },
+     'No floating images to tag, skippping push_floating_tags'),
+    ("called_from_worker",
+     [REGISTRY_V2, OTHER_V2], V2_RESULTS, 'v2',
+     ['namespace/httpd:2.4-1', 'namespace/httpd:latest'],
+     {},
+     'push_floating_tags cannot be used by a worker builder'),
+    ("No_results",
+     [REGISTRY_V2, OTHER_V2], None, 'oci',
+     ['namespace/httpd:2.4-1', 'namespace/httpd:latest'],
+     {
+         'ppc64le': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-ppc64le-latest'],
+         },
+         'x86_64': {
+             REGISTRY_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+             OTHER_V2: ['namespace/httpd:worker-build-x86_64-latest'],
+         }
+     },
+     'No manifest digest available, skipping push_floating_tags'),
+])
+@responses.activate  # noqa
+def test_floating_tags_push(tmpdir, test_name, registries, manifest_results, schema_version,
+                            floating_tags, workers, expected_exception, reactor_config_map,
+                            caplog):
+    if MOCK:
+        mock_docker()
+
+    primary_images = ['namespace/httpd:2.4', 'namespace/httpd:primary']
```

Comment:
nitpick: were the concepts of primary tags and floating tags swapped in the test suite on purpose? A floating tag is the one containing just the `version` while the primary one would be the one with `version-release`.

Reply:
@athos-ribeiro is right, those are examples of floating tags not primary tags. Previous discussion about this line: https://github.com/containerbuildsystem/atomic-reactor/pull/1282#discussion_r345284524

Reply:
The tags are arbitrary for testing purposes.